### PR TITLE
[EDU-4471] refactor: update guides for templates (PT) 

### PR DESCRIPTION
**Pull request by MarianaAguilera**
### Related issue: 

[EDU-4471]

### Changes

Update guides for templates (PT), including:

- Review requirements.
- Align new flow to get and deploy the template.
- Delete manual CD process.

### Additional links

n/a.

[EDU-4471]: https://aziontech.atlassian.net/browse/EDU-4471?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/content/docs/pt-br/pages/guias/marketplace/templates/astro-blog-template.mdx [hunk 1]
```diff
@@ -22,32 +22,16 @@ Este template usa a versão Astro `4.2.6`.
 
 ## Pré-requisitos
 
-Para usar este template, você deve:
+- Uma [conta no GitHub](https://github.com/signup) para conectar com a Azion e criar seu novo repositório.
+  - Cada push será implantado automaticamente neste repositório para manter seu projeto atualizado.
+- Habilitar o [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) e o [Image Processor](/pt-br/documentacao/produtos/build/edge-application/image-processor/) na sua conta Azion.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/astro-blog-template.mdx:27]:
```suggestion
- Habilitar as [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) e o [Image Processor](/pt-br/documentacao/produtos/build/edge-application/image-processor/) na sua conta Azion.
```

Comment (gabriel-azion) [reply src/content/docs/pt-br/pages/guias/marketplace/templates/astro-blog-template.mdx:27]: Talvez o modulo, ou n sei

File: src/content/docs/pt-br/pages/guias/marketplace/templates/astro-blog-template.mdx [hunk 2]
```diff
@@ -22,32 +22,16 @@ Este template usa a versão Astro `4.2.6`.
...
-
-Para habilitar esses módulos:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Active o switch para cada módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch de cada módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/astro-blog-template.mdx:34]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate.mdx [hunk 3]
```diff
@@ -20,16 +20,14 @@ Este boilerplate usa o Astro versão `2.6.4`.
...
+- Habilitar o [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) na sua conta Azion.
 
 Para fazer isso:
 
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/). 
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/astro-boilerplate.mdx:32]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/fauna-template.mdx [hunk 4]
```diff
@@ -23,24 +23,13 @@ Antes de usar este template, você precisa:
...
-<Badge>[Application Accelerator](/pt-br/documentacao/produtos/build/edge-application/application-accelerator/)</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch de cada módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/fauna-template.mdx:34]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx [hunk 5]
```diff
@@ -23,28 +23,18 @@ O Gatsby Boilerplate utiliza a versão `5.13.3` do Gatsby e a versão `18.2.0` d
...
-
-Para habilitar o módulo:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Ative o switch do módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-boilerplate.mdx:34]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx [hunk 6]
```diff
@@ -24,28 +24,18 @@ O Gatsby Boilerplate utiliza a versão `5.13.2` do Gatsby e a versão `18.2.0` d
...
-
-Para habilitar esses módulos:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Ative o switch para cada módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/gatsby-starter-kit.mdx:35]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/hexo-boilerplate.mdx [hunk 7]
```diff
@@ -20,16 +20,13 @@ Este template usa a versão `6.3.0` do Hexo.
...
-</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/hexo-boilerplate.mdx:31]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/mongodb-template.mdx [hunk 8]
```diff
@@ -26,72 +26,57 @@ Antes de usar este template, você precisa:
...
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
-4. Active o switch de cada módulo.
+4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/mongodb-template.mdx:37]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-app-config.mdx [hunk 9]
```diff
@@ -26,29 +26,16 @@ Esse template usa a versão `13.5.6` do Next.
...
 
-Para habilitar esses módulos:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Ative o switch para cada módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch de cada módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-app-config.mdx:38]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-app-middleware.mdx [hunk 10]
```diff
@@ -26,29 +26,16 @@ Esse template usa a versão `13.5.6` do Next.
...
 
-Para habilitar esses módulos:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Ative o switch para cada módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch de cada módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-app-middleware.mdx:38]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-config.mdx [hunk 11]
```diff
@@ -26,29 +26,16 @@ Esse template usa a versão `13.5.6` do Next.
...
 
-Para habilitar esses módulos:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Ative o switch para cada módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch de cada módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-config.mdx:38]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx [hunk 12]
```diff
@@ -26,29 +26,16 @@ Esse template usa a versão `13.5.6` do Next.
...
 
-Para habilitar esses módulos:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Ative o switch para cada módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch de cada módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-pages-middleware.mdx:38]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate.mdx [hunk 13]
```diff
@@ -55,16 +55,13 @@ Esses recursos exigem um servidor Node.js ou uma lógica dinâmica que não pode
...
-</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/next-static-boilerplate.mdx:66]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/qstash-scheduler.mdx [hunk 14]
```diff
@@ -19,26 +19,15 @@ O template **QStash EdgeFunction Scheduler** foi projetado para configurar e ger
...
+- No [console do Upstash](https://console.upstash.com/qstash), obter sua **Current Signing Key**, a **Next Signing Key** e o **QStash Token** na aba **QStash**.
+- Ter uma [conta no GitHub](https://github.com/signup) para conectar com a Azion e criar seu novo repositório.
+  - Cada push será implantado automaticamente neste repositório para manter seu projeto atualizado.
+- Habilitar o [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) na sua conta Azion.
+  - Para fazer isso:
+
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/qstash-scheduler.mdx:32]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/react-boilerplate.mdx [hunk 15]
```diff
@@ -20,16 +20,13 @@ Este template usa o React versão `18.2.0`.
...
-</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/react-boilerplate.mdx:31]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/upstash-geolocation.mdx [hunk 16]
```diff
@@ -21,24 +21,15 @@ Essas mensagens são entregues por meio de uma interface estática, baseada em H
...
+- Criar um banco de dados global para obter a melhor latência de edge disponível no [console do Upstash](https://console.upstash.com/).
+- Ter uma [conta no GitHub](https://github.com/signup) para conectar com a Azion e criar seu novo repositório.
+  - Cada push será implantado automaticamente neste repositório para manter seu projeto atualizado.
+- Habilitar o [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) e o [Application Accelerator](/pt-br/documentacao/produtos/build/edge-application/application-accelerator/) na sua conta Azion.
+  - Para fazer isso:
+
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch de cada módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/upstash-geolocation.mdx:34]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/angular-boilerplate.mdx [hunk 17]
```diff
@@ -20,16 +20,13 @@ Este template usa o Angular CLI versão `16.1.3`.
...
-</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/angular-boilerplate.mdx:31]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/edge-function-github-autodeploy.mdx [hunk 18]
```diff
@@ -20,16 +20,13 @@ Ao usar este template, você só precisa criar sua edge function junto com uma a
...
-</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/edge-function-github-autodeploy.mdx:31]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/hello-world.mdx [hunk 19]
```diff
@@ -18,16 +18,13 @@ O template **Hello World** da Azion cria uma edge application que exibe uma mens
...
-</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/hello-world.mdx:29]:
```suggestion
4. Ative o switch do módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/upstash-rate-limiting.mdx [hunk 20]
```diff
@@ -21,81 +21,65 @@ ___
...
+- Criar um banco de dados global com a melhor latência de edge disponível no [Upstash Console](https://console.upstash.com/).
+- Ter uma [conta no GitHub](https://github.com/signup) para conectar com a Azion e criar seu novo repositório.
+  - Cada push será implantado automaticamente neste repositório para manter seu projeto atualizado.
+- Habilitar o [Edge Functions](/pt-br/documentacao/produtos/build/edge-application/edge-functions/) na sua conta Azion.
+  - Para fazer isso:
+
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (gabriel-azion) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/upstash-rate-limiting.mdx:34]:
```suggestion
4. Ative o switch do módulo.
```

**Review by gabriel-azion — APPROVED** [review]

File: src/content/docs/pt-br/pages/guias/marketplace/templates/vite-boilerplate.mdx [hunk 21]
```diff
@@ -24,29 +24,15 @@ Usa a versão `3.3.4` de Vue e a versão `4.4.6` de Vite.
...
-
-Para habilitar esses módulos:
-
-1. Acesse o [Azion Console](/pt-br/documentacao/produtos/guias/como-acessar-o-azion-console/) > **Account menu**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
 2. Selecione a opção **Billing & Subscriptions**.
-3. Selecione a aba **Subscriptions**.
-4. Ative o switch para cada módulo.
+3. Clique na aba **Subscriptions**.
+4. Active o switch de cada módulo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/vite-boilerplate.mdx:35]:
```suggestion
4. Ative o switch de cada módulo.
```

File: src/content/docs/pt-br/pages/guias/marketplace/templates/vue-boilerplate.mdx [hunk 22]
```diff
@@ -20,16 +20,13 @@ Este template usa o Vue versão `3.3.4`.
...
-</Badge>
-
-Para fazer isso:
-
-1. Acesse o [Azion Console](/en/documentation/products/guides/how-to-access-rtm/).
-2. Selecione **Account menu** > **Billing & Subscriptions**.
+1. Acesse o [Azion Console](https://console.azion.com/) > **Account menu**.
+2. Selecione a opção **Billing & Subscriptions**.
 3. Clique na aba **Subscriptions**.
 4. Active o switch do módulo.
```
Comment (MarianaAguilera) [inline src/content/docs/pt-br/pages/guias/marketplace/templates/vue-boilerplate.mdx:31]:
```suggestion
4. Ative o switch do módulo.
```

**Review by GabrielAzion — APPROVED** [review]

**Pull request merged**